src/anshuKJun20/ExamCode4.java

```diff
@@ -0,0 +1,31 @@
+package anshuKJun20;
+public class ExamCode4 {
+	public static void main(String[] args) {
+		int[] arr = { 10, 33, 55, 43, 99, 21 };
+		int[] arr1 = { 100, 105, 1, 99, 32, 105 };
+		int[] arr2 = { 100, 21, 99, 55, 43 };
+		ExamCode4 exammCode = new ExamCode4();
+		exammCode.findSecondMaximumNumber(arr);
+		exammCode.findSecondMaximumNumber(arr1);
+		exammCode.findSecondMaximumNumber(arr2);
+
+	}
+
+	void findSecondMaximumNumber(int[] arr) {
+		int secMaxNum = 0, maxNum = arr[0];
```

Comment:
here you should define secMax as a[1] and start from indexing from 2
Rest logic is correct